lib/jirawatch/cli/track.rb

```diff
@@ -6,55 +6,84 @@ class Track < Dry::CLI::Command
       include Jirawatch::CLI::AuthenticatedCommand
 
       argument :issue_key, required: true, desc: "Issue key that you want to track time for"
-      option :started_at, aliases: ["-t"], desc: "Specify when you started working on this issue with a HH:mm format"
+      option :time_started_at, aliases: ["-t"], desc: "Specify when you started working on this issue with a HH:mm format"
       option :worklog_message, aliases: ["-m"], desc: "Specify a work log message"
 
       def call(issue_key:, **options)
         @jira_client.Issue.find(issue_key) # Fails if issue doesn't exist
 
+        time_started_at = Time.parse(options.fetch(:time_started_at, Time.now.to_s))
```

Comment:
You're not starting time, you're starting a `timer` 😄 

On second thought, maybe `tracking` is better here instead of `timer` because timer is something that goes from something to zero, while here this is going from zero to something

Reply:
`Timer` keyword can be misunderstanding! I approve the use of the `tracking`/`tracking time` keywords.